src/api.cc

```diff
@@ -404,6 +404,13 @@ class LiteInstall : public InstallContext {
   InstallResult Install() override {
     client_->logTarget("Installing: ", *target_);
 
+    // Call appsInSync to update applications list inside the package manager
+    client_->appsInSync(*target_);
+    client_->setAppsNotChecked();
```

Comment:
What is the use-case `setAppsNotChecked()` is needed for?

Reply:
We need to reset `are_apps_checked_ = false` in case the same instance is used for Downloading a different target. If `are_apps_checked_ == true`, the apps list will not be updated. I'll add a comment in this location, since it is not obvious.

Reply:
ok, maybe it makes sense to add a unit test for this subtle use-case then.

Reply:
There is already a unit test that fails without it inside `test_cli`, at this point:
```
  ASSERT_EQ(cli::Install(*akclient, target01.Version()), cli::StatusCode::Ok);
  ASSERT_EQ(akclient->GetCurrent(), target01);

  auto app01 = registry.addApp(
      fixtures::ComposeApp::create("app-01", "service-01", "image-01", fixtures::ComposeApp::ServiceTemplate,
                                   Docker::ComposeAppEngine::ComposeFile, "compose-start-failure"));
  const auto target02 = createTufTarget(&app01);

  ASSERT_EQ(cli::Install(*akclient, target02.Version()), cli::StatusCode::InstallNeedsReboot);
```

Reply:
ok